Altis_Life.Altis/core/shops/fn_clothingMenu.sqf

```diff
@@ -133,6 +133,10 @@ if (LIFE_SETTINGS(getNumber,"clothing_noTP") isEqualTo 1) then {
     };
 
     player setBehaviour "SAFE";
+    if(_shop == "dive") then {
```

Comment:
Missing a white space after the **if**.
And i think you should change **==** to  **isEqualTo**.